src/reconstruction/rBioNet/data2model.m

```diff
@@ -147,7 +148,7 @@
 %
 
 % if reaction and metabolite database is input manually instead of through rBioNet GUI
-if nargin>2
+if isfield(database,'reactions') && isfield(database,'metabolites')
```

Comment:
You will still need to test, whether database actually exists
i.e.
```
if exist('database','var') && isfield(database,'reactions') && isfield(database,'metabolites')
...
```

Reply:
Ok, fine.